### PR TITLE
fix(chatgpt): inline style override

### LINKED ISSUE #904: unstyled inline styles
### Is there an existing issue outlining your problem?

- [X] I am using the latest version of the provided userstyle.
- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:chatgpt

### Describe your problem.

Sometimes chatgpt decides to do some thing that makes there be inline styles and those inline styles arent the correct color

fix:
```css
[style*="color: rgb(204, 0, 0)"] {
  color: @red;
}
```

### Attach screenshots.

image

image

### What browser(s) are you seeing the problem on?

firefox stable latest

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by trinkey**
## 🔧 What does this fix? 🔧
fixes #904 

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by trinkey** [comment]
i hate github mobile give me a sec

**Comment by uncenter** [comment]
You need to use `closes #<issue>` for GitHub to detect it and automatically close the associated issue.

File: styles/chatgpt/catppuccin.user.css [hunk 1]
```diff
@@ -859,6 +859,10 @@
       color: darken(@yellow, 30%);
     }
 
+    [style*="color: rgb(204, 0, 0)"] {
+      color: @red;
+    }
+
```
Comment (uncenter) [inline styles/chatgpt/catppuccin.user.css:865]: Probably best to put this in the "Other" section below.

**Comment by trinkey** [comment]
> You need to use `closes #<issue>`

i need to start sleeping more i've been so forgetful today

File: styles/chatgpt/catppuccin.user.css [hunk 2]
```diff
@@ -931,6 +931,10 @@
       --tw-ring-offset-color: @base;
     }
 
+    [style*="color: rgb(204, 0, 0)"] {
```
Comment (uncenter) [inline styles/chatgpt/catppuccin.user.css:934]: Can we add a comment here to help us remember what this styles? From your screenshot in the original issue it looked like LaTeX math expressions or something?

**Comment by isabelroses** [comment]
> You need to use `closes #<issue>` for GitHub to detect it and automatically close the associated issue.

if its listed on the right under development, then GitHub is aware of it and can close it.

**Comment by uncenter** [comment]
> > You need to use `closes #<issue>` for GitHub to detect it and automatically close the associated issue.
> 
> if its listed on the right under development, then GitHub is aware of it and can close it.

GitHub knows it is related/mentioned but won't auto close unless the specific keyword is used (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

File: styles/chatgpt/catppuccin.user.css [hunk 3]
```diff
@@ -931,6 +931,11 @@
       --tw-ring-offset-color: @base;
     }
 
+    /* Gives themed color to invalid LaTeX expressions */
```
Comment (uncenter) [inline styles/chatgpt/catppuccin.user.css:934]:
```suggestion
    /* Invalid LaTeX expressions */
```

**Comment by isabelroses** [comment]
> > > You need to use `closes #<issue>` for GitHub to detect it and automatically close the associated issue.
> > 
> > 
> > if its listed on the right under development, then GitHub is aware of it and can close it.
> 
> GitHub knows it is related/mentioned but won't auto close unless the specific keyword is used ([docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)).

Scroll a bit down the the manually adding from sidebar section.

**Comment by uncenter** [comment]
> Scroll a bit down the the manually adding from sidebar section.

Yeah but that is manually???

**Pull request merged**